src/Components/NavBar/NavItem.tsx

```diff
@@ -109,7 +108,6 @@ export const NavItem: React.FC<NavItemProps> = ({
 
 const MenuContainer = styled(Box)`
   position: absolute;
-  top: ${NavBarHeight}px;
```

Comment:
`top` was already being set by https://github.com/artsy/reaction/pull/2951/commits/38c58cf8335e1606a1b1babd07e758bfdfc6e998#diff-298555bc0dcd6937d9db0a76325953adR98 (`<MenuContainer top={space(6)}>`) and checked it in story book and confirmed that the correct `top` is `space(6)` and not `NavBarHeight` which is `space(6) - 1`:

![Screen Shot 2019-11-01 at 1 19 24 PM](https://user-images.githubusercontent.com/687513/68045458-eb612780-fcaf-11e9-9bfe-7f64fe3a9f4b.png)